bridge/gitlab/import.go

```diff
@@ -248,7 +248,14 @@ func (gi *gitlabImporter) ensureNote(repo *cache.RepoCache, b *cache.BugCache, n
 
 		return err
 
-	case NOTE_UNKNOWN:
```

Comment:
`NOTE_UNKNOWN` would lead to a panic

Reply:
Yes agreed that's not a reason to panic. Added `NOTE_UNKNOWN` to the cases